open_spiel/scripts/install.sh

```diff
@@ -119,7 +119,7 @@ function cached_clone() {
 
 DIR="./pybind11"
 if [[ ! -d ${DIR} ]]; then
-  cached_clone -b smart_holder --single-branch --depth 1 https://github.com/pybind/pybind11.git ${DIR}
+  cached_clone -b master --single-branch --depth 1 https://github.com/pybind/pybind11.git ${DIR}
```

Comment:
Very minor: The other cached_clone calls have the arg (master here) in quotes.

Not sure if consistency matters here.